server/main.lua

```diff
@@ -263,7 +255,7 @@ QBCore.Functions.CreateCallback('qb-phone:server:PayInvoice', function(source, c
     local SenderPly = QBCore.Functions.GetPlayerByCitizenId(sendercitizenid)
     local invoiceMailData = {}
     if SenderPly and Config.BillingCommissions[society] then
-        local commission = round(amount * Config.BillingCommissions[society])
+        local commission = QBCore.QBShared.Round(amount * Config.BillingCommissions[society])
```

Comment:
```suggestion
        local commission = QBCore.Shared.Round(amount * Config.BillingCommissions[society])
```
`QBCore.QBShared` doesn't exist, use `QBCore.Shared` instead.